apps/emqx_bridge/src/emqx_bridge_v2_api.erl

```diff
@@ -247,6 +254,34 @@ schema("/actions/:id") ->
             }
         }
     };
+schema("/actions/:id/metrics") ->
+    #{
+        'operationId' => '/actions/:id/metrics',
+        get => #{
+            tags => [<<"actions">>],
+            summary => <<"Get action metrics">>,
+            description => ?DESC("desc_bridge_metrics"),
+            parameters => [param_path_id()],
+            responses => #{
+                200 => emqx_bridge_schema:metrics_fields(),
+                404 => error_schema('NOT_FOUND', "Action not found")
+            }
+        }
+    };
+schema("/actions/:id/metrics/reset") ->
```

Comment:
it was quite controversial in the beginning of v5 api designs: if we should include verbs in the path.

I personally don't think it's really care much, but I'd like to get your opinion on this:
Does it make sense to use DELEET method here instead ?